odoo/addons/base/i18n/vi.po

```diff
@@ -15168,7 +15168,7 @@ msgstr "Phân cách thập phân"
 msgid ""
 "Decimal places taken into account for operations on amounts in this "
 "currency. It is determined by the rounding factor."
-msgstr ""
+msgstr "Vị trí số thập phân được tính đến cho các thao tác trên số tiền bằng đơn vị tiền tệ này. Nó được xác định bởi hệ số làm tròn."
```

Comment:
Google Dịch hả :rofl: 
Cũng nên xuống dòng như nguyên gốc ở `msgid`.
```suggestion
msgstr "Số chữ số thập phân được sử dụng tính toán số tiền của tiền tệ này."
" Nó được tính toán tự động từ Hệ số làm tròn."
```
Trong tiếng Anh thì "Operation" có nhiều nghĩa::
1. Hoạt động: đây là nghĩa thông dụng trong đời sống
2. Phép toán: là nghĩa trong toán học, lập trình. Trong các tài liệu lập trình em có thể thấy thêm các thuật ngữ liên quan như:
   1. operator (có nghĩa là "toán tử", không phải là "người vận hành" như nghĩa thông thường)
   2. operand: toán hạng (các thành phần của 2 bên toán tử gọi là toán hạng)

Em nên mở giao diện phần mềm ra và chuyển dịch để làm sao **người Việt đọc tiếng Việt phải hiểu được**. Google Dịch có thể dùng để tham khảo chứ đừng copy.

Nên kết hợp với từ điển để tra cứu. Mỗi lần tra em hãy cố ghi nhớ thì tiếng Anh sẽ tiến bộ. Google Dịch không làm chúng ta tiến bộ được.

Reply:
@davidtranhp Dạ vâng anh. Em rút kinh nghiệm ạ !